crates/sui-indexer-alt-graphql/src/api/types/event/filter.rs

```diff
@@ -53,21 +265,3 @@ pub(super) fn tx_ev_bounds(
 
     ev_lo..ev_hi
 }
-
-/// The transaction sequence number bounds with pagination cursors applied inclusively.
-pub(super) fn pg_tx_bounds(
-    page: &Page<CEvent>,
-    tx_bounds: std::ops::Range<u64>,
-) -> std::ops::Range<u64> {
-    let pg_lo = page
-        .after()
-        .map(|c| c.tx_sequence_number)
-        .map_or(tx_bounds.start, |tx_lo| tx_lo.max(tx_bounds.start));
-
-    let pg_hi = page
-        .before()
-        .map(|c| c.tx_sequence_number.saturating_add(1))
-        .map_or(tx_bounds.end, |tx_hi| tx_hi.min(tx_bounds.end));
-
-    pg_lo..pg_hi
-}
```

Comment:
Moved this to be done in the sql query to get tx_lo and tx_hi from checkpoint bounds. [tx_bounds_query](https://github.com/MystenLabs/sui/pull/23374/files#diff-152a91c6e6e50627ed8ae880be6607507fb3b76cb30ae7063e3c12d47bfbc398R139)